app/views/content/is-teaching-right-for-me/physics/_article.html.erb

```diff
@@ -0,0 +1,119 @@
+<div class="row inset">
+<section class="col col-720">
+  <p>
+    Teaching physics is an exciting and rewarding career. Schools need more specialist physics teachers, so it's a great choice of subject.
+  </p>
+
+  <p>
+You could inspire your pupils to consider rewarding careers from tackling climate change to artificial intelligence (AI).
+  </p>
+  <p>Tax-free bursaries of £28,000 or scholarships of £30,000 are available for eligible trainee physics teachers.</p>
+
+  </section>
+
+   <section class="col col-space-s col-content-max">
+    <%= render TextBlockWithTwoImagesComponent.new(
+      heading: "Spark curiosity in physics",
+      colour: "yellow",
+      image_1_path: "static/images/content/hero-images/0012.jpg",
+      image_1_alt: "",
+      image_2_path: "static/images/content/hero-images/0007.jpg",
+      image_2_alt: "",
+    ) do %>
+       <p>As a physics teacher, you’ll be engaging students in a wide range of experiments that can spark their interest.</p>
+
+    <% end %>
+</section>
+<section class="col col-720">
+      <h2>What you'll be teaching</h2>
+        <section class="col col-720">
+  <%= render Content::QuoteComponent.new(
+    text: "I’m always showing my students how science relates to everything they’re learning. From enzymes in cooking to the surprising history of a lot of medical discoveries. I love it when they ask a question that I’ve never even thought of.",
```

Comment:
@caitlin-reidy think this is a good quote for general science but not sure it's necessarily relevant to physics as the examples she gives sound more like biology or chemistry?